modules/cnv-connecting-resource-bridge-network.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv-attaching-vm-multiple-networks.adoc
+
+[id="connecting-resource-bridge-network-{context}"]
+= Connecting a resource to a bridge-based network
+
+As a network administrator, you can configure a NetworkAttachmentDefinition
+of type `cnv-bridge` to provide Layer-2 networking to Pods and virtual machines.
+
+.Prerequisites
+
+* {ProductName} 2.0 or newer
+* A Linux bridge must be configured and attached to the correct
+Network Interface Card on every node.
+* `vlan_filtering` must be enabled on the bridge if you want to use VLANs.
```

Comment:
the user also need to add this line for every vlan

`
bridge vlan add dev <external interface> vid <vlan-id> master
`
example:
`
bridge vlan add dev bond0 vid 2 master
`

Reply:
Can you explain this in plainer language? Should this be a step in the procedure, a prerequisite, or its own procedure? Thanks.

Reply:
this should be here with the vlan_filtering.

what this line does it's allow pod/vms connected to the bridge with a specific vlan to exit the host using the host interface connected to the bridge.